diff --git a/node_modules/cellar-js-sdk/dist/cellar.js b/node_modules/cellar-js-sdk/dist/cellar.js
index e62cc8f..7b66388 100644
--- a/node_modules/cellar-js-sdk/dist/cellar.js
+++ b/node_modules/cellar-js-sdk/dist/cellar.js
@@ -9,7 +9,8 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
 };
 import { CellarWindow } from "./iframe";
 import { CellarEnv } from "./interface/cellar";
-const pJson = require('../package.json');
+import pJson from '../package.json';
+
 export class Cellar {
     constructor({ appId, env }) {
         this.isLogin = false;
